modules/layers/src/scatterplot-layer/scatterplot-layer.js

```diff
@@ -30,18 +30,57 @@ const DEFAULT_COLOR = [0, 0, 0, 255];
 
 const defaultProps = {
   radiusScale: {type: 'number', min: 0, value: 1},
-  radiusMinPixels: {type: 'number', min: 0, value: 0}, //  min point radius in pixels
-  radiusMaxPixels: {type: 'number', min: 0, value: Number.MAX_SAFE_INTEGER}, // max point radius in pixels
-  strokeWidth: {type: 'number', min: 0, value: 1},
-  outline: false,
+  radiusMinPixels: {type: 'number', min: 1, value: 1}, //  min point radius in pixels
+  radiusMaxPixels: {type: 'number', min: 1, value: Number.MAX_SAFE_INTEGER}, // max point radius in pixels
+  lineWidthMinPixels: {type: 'number', min: 0, value: 1},
+  lineWidthMaxPixels: {type: 'number', min: 0, value: Number.MAX_SAFE_INTEGER},
```

Comment:
Where are these used?

Reply:
You are not doing anything to restrict the min/max pixels of the line width. Test it in the layer browser.